plugins/json-logger/README.md

```diff
@@ -0,0 +1,27 @@
+# @amplication/plugin-<name>
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-basic)](https://www.npmjs.com/package/@amplication/plugin-auth-basic)
+
+A short description of the plugin and its actions.
+
+## Purpose
+
+What is the purpose of this plugin and what exactly does it do.
+
+## Configuration
+
+If a configuration is required, add it here.
+
+## Scripts
+
+### `build`
+
+Running `npm run build` will bundle your plugin with Webpack for production.
+
+### `dev`
+
+Running `npm run dev` will watch your plugin's source code and automatically bundle it with every change.
+
+## Usage
+
+Explain the usage of this plugin and its effect on the final build.
```

Comment:
Lastly please provide the result this plugin will have on the generate code.

Reply:
![screen](https://github.com/amplication/plugins/assets/123747762/ef96da9e-87c6-479a-8454-343a2d904a2c)
is this kind of logs correct (i mean the form of it , above are the nest logs and below is the custom ones .P.S certain fields are need to be changed to like service name and additional setting etc)